.travis.yml

```diff
@@ -18,6 +18,8 @@ before_cache:
 
 matrix:
   include:
+    - compiler: "ghc-8.8.1"
```

Comment:
Can we drop the `ghc-8.2.2` stanza in here, since we’ve decided to officially support N-2 versions?

Reply:
I don’t think we officially decided anything in #185 😊 More to the point, I’d like to keep supporting 8.2 until we have a pressing reason to change (e.g. avoiding `CPP` 😆).

Reply:
Further, dropping 8.2 would mean a major version bump, IMO.